spec/system/orders_spec.rb

```diff
@@ -32,5 +35,16 @@
       visit orders_path
       expect(page).not_to have_content('なし')
     end
+
+    it '注文詳細ページを閲覧できる' do
+      create :order, user: hakjae
+      visit orders_path
+      within '.table' do
+        find('a').click
+      end
```

Comment:
リンクになっているところに `order` というクラスが付与されていれば、
```suggestion
      click_on '.order', match: :first
```
こう書ける。

`find('a')` は要素が増減したときにテストが壊れやすいので注意

Reply:
確認したかどうかがわからないので、リアクションとか入れてもらえると助かる。

Reply:
確認して修正忘れてました💦